### DISCUSSION TITLE
865531D9-8EBF-4F62-AA7C-26A2FFA78892

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#173] [General]
# Kepler-705

Kepler-705, also known as KOI-854, is a binary star system located 901.5315 light-years away from us. In 2016, a planet larger than Earth was discovered in this system, with a orbital period of 56 Earth days. Considering its appropriate distance from the star, it may be a potential habitable planet

https://celestia.mobi/addon?item=865531D9-8EBF-4F62-AA7C-26A2FFA78892

**Comment by exocritic5457** [comment]
That is Venus. You are not fooling anyone.

**Reply by exocritic5457** [reply]
even still, if I ignore that this is just Venus orbiting a star an extremely specific amount of light-years away, what is with this trend of making planets that likely aren't habitable so obnoxiously Earthlike?

**Reply by gtfggjbdrff** [reply]
The texture of this planet was created using the texture batch of Venus, but the author of this plugin is really working hard. He is not trying to gain downloads, nor is he deceiving anyone. I have learned about the behind the scenes story of this plugin. He created this plugin for the Celestia in Baidu Tieba to plug into the planet system's planet insertion Sisyphus project. Due to its urgency, this plugin can only be used to make ends meet

**Reply by gtfggjbdrff** [reply]
Please don't criticize him, he's not lying, I don't mean to protect him because it's true

**Reply by gtfggjbdrff** [reply]
The author of this plugin encountered many difficulties in the Sisyphus project of the planetary system

**Reply by TyphoonRagasa** [reply]
The data for this planet comes from Wikipedia's list of potentially habitable exoplanets, so it is made as an Earth-like planet rather than Venus. In addition, this newly updated add-on also includes a sub-Neptune texture.

**Comment by FaithfulCocoa** [comment]
in all seriousness, the texture map for this addon is literally terraformed venus, instead of actually going through effort to make the texture, they grabbed a terraformed venus and called it a day

**Reply by DaveBowman2001** [reply]
@FaithfulCocoa please no mockery comments next time.

**Comment by gtfggjbdrff** [comment]
The author of this plugin has already worked very hard. Please do not post sarcastic comments or directly criticize. Thank you

**Reply by FaithfulCocoa** [reply]
having an opinion on something is a crime????

**Reply by gtfggjbdrff** [reply]
The texture of this planet was created using the texture batch of Venus, but the author of this plugin is really working hard. He is not trying to gain downloads, nor is he deceiving anyone. I have learned about the behind the scenes story of this plugin. He created this plugin for the Celestia in Baidu Tieba to plug into the planet system's planet insertion Sisyphus project. Due to its urgency, this plugin can only be used to make ends meet

**Reply by gtfggjbdrff** [reply]
Please do not spray each other, thank you

**Reply by FaithfulCocoa** [reply]
thanks for the explanation
